tests/test_metrics/test_metrics_utils.py

```diff
@@ -172,10 +103,6 @@ def aggregated_metrics_df_with_full_missing() -> Dict[str, Any]:
     [
         ("metrics_df_with_folds", "aggregated_metrics_df"),
         ("metrics_df_no_folds", "aggregated_metrics_df"),
-        ("metrics_df_with_folds_with_missing", "aggregated_metrics_df_with_missing"),
-        ("metrics_df_no_folds_with_missing", "aggregated_metrics_df_with_missing"),
-        ("metrics_df_with_folds_with_full_missing", "aggregated_metrics_df_with_full_missing"),
-        ("metrics_df_no_folds_with_full_missing", "aggregated_metrics_df_with_full_missing"),
```

Comment:
Why did you remove these tests?

Reply:
I just rearranged the tests. I put all of the possible cases in a single fixture.